run.py

```diff
@@ -0,0 +1,172 @@
+from flask import render_template, request, jsonify, Flask
+import os, re, datetime
+from app import db
+from app.product.models import ProductModel
+
+
+app = Flask(__name__)
+BASE_URL = '/products'
+
+if not os.path.isfile('products.db'):
+    db.connect()
+
+def isValid(email):
+    regex = re.compile(r'([A-Za-z0-9]+[.-_])*[A-Za-z0-9]+@[A-Za-z0-9-]+(\.[A-Z|a-z]{2,})+')
+    if re.fullmatch(regex, email):
+      return True
+    else:
+      return False
```

Comment:
wystarczy, że zwrócisz `return re.fullmatch(regex, email)`. To sprawdzenie za pomocą `if` i `else` nie jest potrzebne